README.md

```diff
@@ -2,6 +2,6 @@
 
 This example follows the instructions of the [Getting Started](https://docs.devexpress.com/WPF/5863/controls-and-libraries/data-grid/getting-started) tutorial. It demonstrates the [WPF Data Grid](https://docs.devexpress.com/WPF/6084/controls-and-libraries/data-grid) base functionality.
```

Comment:
"This example follows the instructions" to do what?

Reply:
When you write a noun string like "WPF Data Grid base functionality", it means it all is one thing, which is incorrect.